packages/replayio/src/utils/browser/launchBrowser.ts

```diff
@@ -28,9 +29,10 @@ export async function launchBrowser(
   ];
   const processOptions = {
     env: {
-      RECORD_ALL_CONTENT: "1",
+      RECORD_ALL_CONTENT: record ? "1" : undefined,
+      RECORD_REPLAY_DONT_RECORD: record ? undefined : "1",
```

Comment:
TIL, what is this? test plugins dont set this at all (IIRC)

Reply:
sideways: do u know what `RECORD_REPLAY_DRIVER` is for? test plugins use it but we don't use it here

Reply:
TIL too. Jason mentioned it in Discord. All I know about it is that if it's set, it tells the runtime not to record anything.

Reply:
> sideways: do u know what `RECORD_REPLAY_DRIVER` is for? test plugins use it but we don't use it here

No. Maybe we should remove it too? I don't remember why it's here. :)